cookbooks/kibana/attributes/default.rb

```diff
@@ -1,4 +1,5 @@
 # Attributes for Kibana
 
-default["kibana"]["dir"]   = "/home/#{node["delk_user"]}/kibana"
-default["kibana"]["port"]  = "5601"
+default["kibana"]["dir"]      = "/home/#{node["delk_user"]}/kibana"
+default["kibana"]["port"]     = "5601"
+default["kibana"]["all_dirs"] = %w( . conf )
```

Comment:
I would leave out the dot and use `directory` to set the parent dir's permissions.
You mentioned that something wasn't working with this approach. We can look into it.